presto-main/src/test/java/io/prestosql/sql/planner/TestDynamicFilter.java

```diff
@@ -254,8 +261,8 @@ public void testNestedDynamicFiltersRemoval()
                                 INNER,
                                 ImmutableList.of(equiJoinClause("ORDERS_CK", "ORDERS_CK6")),
                                 ImmutableMap.of("ORDERS_CK", "ORDERS_CK6"),
-                                Optional.empty(),
```

Comment:
I think this test requires some comment. As I read it now, I'm not sure what nested DF this test is about.

Reply:
This was done to fix https://github.com/prestosql/presto/issues/2355
We made changes to remove DFs which became part of nested expressions due to predicate pushdown.

Reply:
How does it become part of nested expression? I don't see this.